tox.ini

```diff
@@ -19,7 +19,7 @@ commands =
     {toxinidir}/superset/bin/superset db upgrade
     {toxinidir}/superset/bin/superset init
     nosetests tests/load_examples_test.py
-    nosetests -e load_examples_test tests {posargs}
+    nosetests --exclude=load_examples_test {posargs:tests}
```

Comment:
`-e` and `--exclude=` are equivalent, though I though `--exclude` was more descriptive.